apollo-client/src/main/java/com/ctrip/framework/apollo/util/ConfigUtil.java

```diff
@@ -509,4 +509,11 @@ private boolean getPropertyBoolean(String propertyName, String envName, boolean
     }
     return defaultVal;
   }
+
+  public String getAppSecret(String appId){
+    if(appId.equals(getAppId())){
+      return getAccessKeySecret();
+    }
+    return System.getProperty("apollo.accesskey." + appId + ".secret", "");
```

Comment:
It's better to add the logic to `Foundation.app().getAccessKeySecret`, which has a comprehensive list of places to check.